includes/Admin/Editor.php

```diff
@@ -230,18 +217,16 @@ public function admin_enqueue_scripts( $hook ) {
 	 * @since 1.0.0
 	 *
 	 * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
-	 *
-	 * @return array
 	 */
-	public function get_editor_settings() {
+	public function get_editor_settings(): array {
 		$post                     = get_post();
-		$story_id                 = ( $post ) ? $post->ID : null;
+		$story_id                 = ( $post ) ? (int) $post->ID : null;
 		$rest_base                = $this->get_post_type_rest_base( Story_Post_Type::POST_TYPE_SLUG );
 		$has_publish_action       = $this->get_post_type_cap( Story_Post_Type::POST_TYPE_SLUG, 'publish_posts' );
 		$has_assign_author_action = $this->get_post_type_cap( Story_Post_Type::POST_TYPE_SLUG, 'edit_others_posts' );
 		$has_upload_media_action  = current_user_can( 'upload_files' );
 
-		if ( $story_id ) {
+		if ( $story_id && is_int( $story_id ) ) {
```

Comment:
is that `is_int()` check needed if the type is coerced to an `(int)` above? 🤔

Reply:
Just being lovely careful, setup_lock needs a int now.

Reply:
But it's already made an `int` above, so it would never be _not_ an int.
And `if ( $story_id )` catches the `null` case.

Reply:
Fixed.